lib/elixir/lib/kernel/utils.ex

```diff
@@ -189,7 +189,7 @@ defmodule Kernel.Utils do
   that checks for its presence in a guard, then unquotes the variable references as
   appropriate.
 
-  The following code
+  The following code:
```

Comment:
No colon here because the expression continues after the example. :)

```suggestion
  The following code
```